docs/current_docs/manuals/developer/cookbook/snippets/secret-file/go/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"context"
+)
+
+type MyModule struct{}
+
+// Query the GitHub API
+func (m *MyModule) GithubApi(
```

Comment:
```suggestion
func (m *MyModule) GithubAuth(
```
to match function call via cli